// userData/userData.js
var mysql = require('mysql');
var result = {};
let test = 0;

exports.getUser = function(userId) {
	// var result = {};
	var connection = mysql.createConnection({
		host: 'localhost',
		port: 3306,
		user: 'root',
		password: '123',
		database: 'testdemoproject'
	})
	connection.connect();
	
	connection.query('select * from users where id =' + userId,function(error, results) {
		result.userMessage = results;
	})
	
	connection.end();
	return result;
}


// exports.getUser = function(userId) {
// 	var resultset = {};
// 	var connection = mysql.createConnection({
// 		host: 'localhost',
// 		port: 3306,
// 		user: 'root',
// 		password: '123',
// 		database: 'testdemoproject'
// 	})
// 	connection.connect();
// 	
// 	connection.query('select * from users where id =' + userId, function(error, results) {
// 		// console.log(results);
// 		if(error) {
// 			console.log(error, "数据库读取出错");
// 		}
// 		resultset.userMessage = results;
// 		// console.log(result)
// 	})
// 	resultset.test = test;
// 	console.log(resultset)
// 	connection.end();
// 	return resultset;
// }
// 
// exports.out = function() {
// 	console.log('Test');
// 	return result;
// }

// exports.getData = function(page) {
// 	var connection = mysql.createConnection({
// 		host: 'localhost',
// 		port: 3306,
// 		user: 'root',
// 		password: '123',
// 		database: 'testdemoproject'
// 	})
// 	connection.connect();
// 	
// 	connection.query('select * from users',function(error, results) {
// 		result.resultset = results;
// 		console.log('inner', results)// 查询正确
// 		// 数据库查询异步
// 		// 
// 	})
// 	console.log('--------------------------------')
// 	connection.end();
// 	return result;
// }
